python/pip_install/extract_wheels/lib/bazel.py

```diff
@@ -143,13 +148,15 @@ def generate_build_file_contents(
             # search path for anything that depends on this.
             imports = ["."],
             deps = [{dependencies}],
+            tags = [{tags}],
         )
         """.format(
             name=name,
             dependencies=",".join(dependencies),
             data_exclude=json.dumps(data_exclude),
             whl_file_label=WHEEL_FILE_LABEL,
             whl_file_deps=",".join(whl_file_deps),
+            tags=",".join(tags),
```

Comment:
You could consider inlining the tags definition here as it appears to be unused elsewhere.

Reply:
actually the tags really want to come from the calling function